src/Http/Client/Adapter/Curl.php

```diff
@@ -149,7 +149,7 @@ public function buildOptions(RequestInterface $request, array $options): array
             $out[CURLOPT_PROXY] = $options['proxy']['proxy'];
         }
         if (isset($options['proxy']['username'])) {
-            $password = !empty($options['proxy']['password']) ? $options['proxy']['password'] : '';
+            $password = empty($options['proxy']['password']) ? '' : $options['proxy']['password'];
```

Comment:
Why are we reverting positive !empty() again?
I thought this one was deactivated for this specific one?

Reply:
I will revert this, i forgot to add it into ignore list